tests/dataSources_test.go

```diff
@@ -857,7 +859,7 @@ var _ = Describe("DataSources", func() {
 					logObject(dataImportCron.GetKey(), &cdiv1beta1.DataImportCron{})
 				})
 
-				It("[test_id:8112] should restore DataSource if DataImportCron removed from SSP CR", func() {
+				It("[QUARANTINE][test_id:8112] should restore DataSource if DataImportCron removed from SSP CR", func() {
```

Comment:
I'm not sure `[QUARANTINE]` has any effect on tests in ssp-operator. You can mark the test as pending to disable it (use `PIt()`).